### ISSUE TITLE
Support different commands

### ISSUE THREAD

**Issue by fubuloubu** [ApeWorX/github-action#1]
Allow the end user of this in actions to call differnet commands like `ape compile --size` and `ape test` (and maybe `ape run my_script`?)

**Comment by fubuloubu** [comment]
One possible solution, user adds this to their `.github/workflow/action.yaml` file:

```yaml
- uses: ApeWorX/github-action@main
  with:
    command: compile --size
- uses: ApeWorX/github-action@main  # Uses `ape test` by default
```

**Comment by fubuloubu** [comment]
solved by #4
